recipes-bsp/u-boot/files/0010-dts-iot2050-Support-new-IOT2050-SM-variant.patch

```diff
@@ -0,0 +1,447 @@
+From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
+From: Baocheng Su <baocheng.su@siemens.com>
+Date: Wed, 29 Nov 2023 23:35:51 +0800
+Subject: [PATCH] dts: iot2050: Support new IOT2050-SM variant
+
+the dts file for IOT2050-SM variant is copied from kernel side without
+any change.
```

Comment:
Maybe some words on the new variant here would be nice, similar to the kernel patch.

Reply:
Fixed in #509